test/Object/rlexe.xml

```diff
@@ -450,4 +450,10 @@
       <baseline>assign.baseline</baseline>
     </default>
   </test>
+  <test>
+    <default>
+      <files>ObjectSpread.js</files>
+      <compile-flags>-args summary -endargs -NoNative -ES2018ObjectSpread</compile-flags>
```

Comment:
You have `-ES2018` here while the flag in `ConfigFlagsList.h` is `-ES2019`, which is causing your test to fail due to unknown flags

Reply:
that's strange I thought I changed that part...turns out I didn't commit the change...